cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/VendorReportLineDTO.java

```diff
@@ -6,6 +6,10 @@
 import java.math.BigDecimal;
 import java.util.Objects;
 
+/**
+ * @deprecated by {@link ResellingReportLineDTO}
+ */
+@Deprecated
 public class VendorReportLineDTO extends ReportLineDTO
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)